src/contributors/MAHANTH-wq.md

```diff
@@ -0,0 +1,7 @@
+---
+
+ name: "MAHANTH"
+ username: "MAHANTH-wq"
+ image: "MAHANTH-wq"    // this image name should be the actual name of image you uploaded
```

Comment:
You image has extension of ".png". So, change "MAHANTH-wq" to "MAHANTH-wq.png"

Reply:
Yeah, @MAHANTH-wq Thanks for your contribution small changes need to be made in our PR, just change the extension of your image inside `src/contributors/MAHANTH-wq.md `  from  `image: "MAHANTH-wq" ` to `image: "MAHANTH-wq.png"` and you can remove the comment as well

Reply:
okay